### DISCUSSION TITLE
MBS outfit codes

### DISCUSSION THREAD

**Discussion by Klinge-outfit** [bananarama92/MBS#12] [Q&A] [answered]
Доброго времени суток, подскажите, как правильно и где создать коды костюмов для колеса фортуны

**Comment by bananarama92** [comment]
Hi there, MBS is an English-based repository and, for the sake of achieving a productive discussion, if you have any questions or comments then I must ask you to do so in English.

**Reply by Klinge-outfit** [reply]
I am asked, how to create outfit code for wheel of fortune)

**Comment by bananarama92** [comment, marked as answer]
Please see https://github.com/bananarama92/MBS/discussions/6#discussioncomment-5362086:
> Hi Wotan, all clothes and restraints are specified via the "outfit code" field, which accepts outfit codes as generated via [FBC](https://sidiousious.gitlab.io/bce/) (via the `/exportlooks` command) or [BCX](https://jomshir98.github.io/bondage-club-extended/) (via the "export outfit" button in the wardrobe). Both approaches will produce an outfit code that can subsequently be copy/pasted into the respective field in MBS.
